spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -289,7 +294,7 @@ def create!(csr: nil, bundle_id: nil)
 
           # look up the app_id by the bundle_id
           if bundle_id
-            app = Spaceship::Portal::App.set_client(client).find(bundle_id)
+            app = portal_type.set_client(client).find(bundle_id)
```

Comment:
maybe I missed it in the code, but where does this come from when it is not a website_push? For that case you added the definition above in line 133 - but for production or development?